src/config.py

```diff
@@ -14,5 +19,12 @@ def update_config():
   with open('config/2019.json', 'r') as config_file:
     config_json['2019'] = json.load(config_file)
 
-
+    for contributor_id, contributor in config_json['2019']['contributors'].items():
+      if 'gravatar' in contributor:
+        gravatar_url = "https://www.gravatar.com/avatar/" + hashlib.md5(contributor['gravatar'].lower().encode()).hexdigest() + ".jpg?"
+        gravatar_url += urllib.parse.urlencode({'d': 'mp','s':str(AVATAR_SIZE)})
+        contributor['avatar_url'] = gravatar_url
+      else:
+        contributor['avatar_url'] = DEFAULT_AVATAR_FOLDER_PATH + str(hash(contributor_id) % AVATARS_NUMBER) + ".jpg"
```

Comment:
IIUC this will produce avatar numbers in the range 0 - 14. The avatar filenames go from 1 - 15. Could you reconcile this either by adding 1 to the modulo output or renaming file 15 to 0?